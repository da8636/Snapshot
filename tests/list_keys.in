SET a 1
SET c 2
SET b 3
LIST KEYS
BYE
