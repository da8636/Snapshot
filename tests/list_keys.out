ok

ok

ok

b
c
a

bye
